web/app.js

```diff
@@ -1378,12 +1374,9 @@ const PDFViewerApplication = {
       }
     }
 
-    if (!this.supportsPrinting) {
-      return;
-    }
     if (triggerAutoPrint) {
-      setTimeout(function () {
-        window.print();
+      setTimeout(() => {
```

Comment:
Note that I'm not actually sure if/why `setTimeout` is (still) necessary here!?

As far as I can tell, this was present already in PR #2839 which originally added this auto-print feature, hence there's probably a good reason for this.